composer.json

```diff
@@ -36,16 +36,16 @@
     "bin": ["bin/check_perms"],
     "require": {
         "symfony/polyfill-iconv": "^1",
-        "phpro/grumphp": "^1",
+        "phpro/grumphp": "^0.18.0",
         "drupal/coder": "^8",
         "phpcompatibility/php-compatibility": "^9.3",
         "pheromone/phpcs-security-audit": "^2.0",
-        "wunderio/ecs-drupal": "^1.0",
-        "symplify/coding-standard": "^8.3",
+        "wunderio/ecs-drupal": "^0.2.0",
```

Comment:
Is this even needed as we also have `mglaman/phpstan-drupal`? ;)

Reply:
I don't see that being pulled in from https://github.com/mglaman/phpstan-drupal/blob/master/composer.json

Need to test what happens if removed.

Reply:
Pretty sure it's needed in 1.x as it was there before https://github.com/wunderio/code-quality/blob/bef94aaa1ecf3a220628bb2e932bc03779d26d8c/composer.json